scripts/pre_commit_linter.py

```diff
@@ -180,6 +186,20 @@
 _MESSAGE_TYPE_FAILED = 'FAILED'
 
 
+def _is_filename_excluded_for_bad_patterns_check(pattern, filename):
+    """Checks if file is excluded from the bad patterns check
+
+    Args:
+    - pattern: The pattern to be checked against.
```

Comment:
The entire file seems to have the format:

Args:
- pattern:
- filename: 

Should I change the other occurences or let it be?